### DISCUSSION TITLE
【求助】让原神添加自选圣遗物

### DISCUSSION THREAD

**Discussion by GWWA** [share121/inter-knot#2163] [其他]
来个勇士黑掉米哈游！！！

**Comment by HifumiMizuhara** [comment]
据不明消息，5.0会增加崩铁那种选主词条的

**Reply by HFOxin** [reply]
好像是一个解包的内鬼说的

**Comment by zzz-Rope** [comment]
预言家已经夺舍策划了<img src="https://img-static.mihoyo.com/communityweb/upload/ea409a97f2be1047494df31837ee5217.gif" alt="草史莱姆-开心~" width="75" >

**Comment by JadenDamitao** [comment]
预言家刀了刀了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by TetsuyaTetsuya** [comment]
预言家来的，真有勇士黑了米哈游吗
